src/test/resources/anmn_timeseries.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0"?>
+<definition>
+  <!-- anmn_timeseries -->
+  <source>
+    <schema>anmn_ts</schema>
+    <virtualDataTable>select ts_id as instance_id, * from measurement</virtualDataTable>
+    <virtualInstanceTable>select * from timeseries</virtualInstanceTable>
+  </source>
+  <dimensions>
+    <dimension name="TIME"/>
+  </dimensions>
+  <globalattributes>
+    <attribute name="toolbox_version" sql="select toolbox_version from timeseries where ..."/> <!-- sql query to be completed with possibly id or ts_id = current_id -->
+    <attribute name="file_version" value="Level 1 - Quality Controlled Data"/>
+    <attribute name="file_version_quality_control" value="Quality controlled data have passed quality assurance procedures such as automated or visual inspection and removal of obvious errors. The data are using standard SI metric units with calibration and other routine pre-processing applied, all time and location values are in absolute coordinates to agreed to standards and datum, metadata exists for the data or for the higher level dataset that the data belongs to. This is the standard IMOS data level and is what should be made available to eMII and to the IMOS community."/>
+    <attribute name="project" value="Integrated Marine Observing System (IMOS)"/>
+    <attribute name="Conventions" value="CF-1.6,IMOS-1.3"/>
+    <attribute name="standard_name_vocabulary" value="CF-1.6"/>
+    <attribute name="date_created" sql="select to_char(now() AT TIME ZONE 'UTC', 'YYYY-MM-DD""T""HH24:MI:SSZ')"/>
+    <attribute name="comment" sql="select comment from timeseries where ..."/> <!-- sql query to be completed with possibly id or ts_id = current_id -->
+    <attribute name="instrument" sql="select instrument from timeseries where ..."/> <!-- sql query to be completed with possibly id or ts_id = current_id -->
+    <attribute name="references" value="http://www.imos.org.au"/>
+    <attribute name="site_code" sql="select site_code from timeseries where ..."/> <!-- sql query to be completed with possibly id or ts_id = current_id -->
```

Comment:
Nice @ggalibert !!  This is exactly what was needed.  

Some minor comments @ggalibert, @julian1 :

Perhaps using an sql content element rather than an attribute would allow the sql make it easier to format the sql for readability

e.g.
 
    <attribute name="site_code">
       <sql>
             select to_char(now() AT TIME ZONE 'UTC', 'YYYY-MM-DD""T""HH24:MI:SSZ')
               from timeseries
              where
       </sql>
    </attribute>